.github/workflows/android.yml

```diff
@@ -0,0 +1,26 @@
+name: Android CI
```

Comment:
Minor: You could make the name more meaningful like a _Verification pull request_ or something else